beheer.md

```diff
@@ -104,13 +104,13 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria##
```

Comment:
```suggestion
## Criteria
```

Reply:
Hihi, dat krijg je met die oude SGML krassers, die maken begin- en sluittags ;-)

Reply:
https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language

1986...

Reply:
<opaVerteld>Weet ik, ik ben er in 1992 mee begonnen &smiley; Het SGML Handbook van Charles Golfarb was onze bijbel.  </opaVerteld>
Off topic maar leuk voor mensen die interesse hebben in het modelleren van alledaagse soorten tekst: de Text Encoding Initiative oftewel TEI (https://en.wikipedia.org/wiki/Text_Encoding_Initiative) was 1 van de dingen waar ik me mee bezig hield. Later kwam daar DSSSL (https://en.wikipedia.org/wiki/Document_Style_Semantics_and_Specification_Language) bij.

Reply:
Dat heb k nog ergens liggen. DSSSL heb ik nooit gedaan wel heel veel DTD's ontworpen.

Reply:
Voor mijn afstuderen heb ik geprobeerd een DSSSL interpreter te bouwen (met lex en yacc, in C). Laat ik het zo zeggen, dat is nog best lastig. De eerste versies waren wel heel goede core-dump generatoren! :-D

Reply:
heren, zie ook mij andere comments ;-)

Reply:
Dat stellen we graag zo lang mogelijk uit ;-)

Reply:
Is aangepast.